clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -940,11 +944,14 @@ static Expected<StringRef> runLLVMToSPIRVTranslation(StringRef File,
 /// options are generated for AOT compilation targeting Intel GPUs.
 static void addBackendOptions(const ArgList &Args,
                               SmallVector<StringRef, 8> &CmdArgs, bool IsCPU) {
-  StringRef OptC =
-      Args.getLastArgValue(OPT_sycl_backend_compile_options_from_image_EQ);
+  StringRef OptC = SYCLCompileOptionsFromImage;
+  OptC.split(CmdArgs, " ", /*MaxSplit=*/-1, /*KeepEmpty=*/false);
+  OptC = Args.getLastArgValue(
+      OPT_sycl_backend_compile_options_EQ); // TODO: test this.
```

Comment:
```suggestion
      OPT_sycl_backend_compile_options_EQ); // TODO: test this.
```
Is the TODO still necessary?